texttospeech/cloud-client/ssml_addresses_test.py

```diff
@@ -39,16 +39,11 @@ def test_ssml_to_audio(capsys):
 
     # Assert audio file generated
     ssml_to_audio(input_ssml, 'test_example.mp3')
-    assert os.path.isfile('test_example.mp3')
-
-    # Assert audio file generated correctly
-    assert filecmp.cmp('test_example.mp3',
-                       'resources/expected_example.mp3',
```

Comment:
Is expected_example.mp3 still needed?

Reply:
None of the tests depend on it, but my thinking was that it's nice to have an MP3 file in the repo that demonstrates what the correct output is.

Reply:
Let's remove it, we try not to have unused files.

Reply:
Sounds good--just removed it.